build.gradle

```diff
@@ -50,163 +54,171 @@ openApi {
     }
-    implementation 'org.springframework.boot:spring-boot-starter-jetty:3.2.4'
-    
-    implementation 'org.springframework.boot:spring-boot-starter-thymeleaf:3.2.4'
+    implementation "org.springframework.boot:spring-boot-starter-jetty:$springBootVersion"
+
+    implementation "org.springframework.boot:spring-boot-starter-thymeleaf:$springBootVersion"
 
-    if (System.getenv('DOCKER_ENABLE_SECURITY') != 'false') {
-        implementation 'org.springframework.boot:spring-boot-starter-security:3.2.4'
-        implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity5:3.1.2.RELEASE'
-        implementation "org.springframework.boot:spring-boot-starter-data-jpa:3.2.4"
-        implementation 'org.springframework.boot:spring-boot-starter-oauth2-client:3.2.4'
+    if (System.getenv("DOCKER_ENABLE_SECURITY") != "false") {
+        implementation "org.springframework.boot:spring-boot-starter-security:$springBootVersion"
+        implementation "org.thymeleaf.extras:thymeleaf-extras-springsecurity5:3.1.2.RELEASE"
+        implementation "org.springframework.boot:spring-boot-starter-data-jpa:$springBootVersion"
+        implementation "org.springframework.boot:spring-boot-starter-oauth2-client:$springBootVersion"
 
         //2.2.x requires rebuild of DB file.. need migration path
         implementation "com.h2database:h2:2.1.214"
     }
 
-    testImplementation 'org.springframework.boot:spring-boot-starter-test:3.2.4'
+    testImplementation "org.springframework.boot:spring-boot-starter-test:$springBootVersion"
 
     // Batik
-    implementation 'org.apache.xmlgraphics:batik-all:1.17'
+    implementation "org.apache.xmlgraphics:batik-all:1.17"
 
     // TwelveMonkeys
-    implementation 'com.twelvemonkeys.imageio:imageio-batik:3.10.1'
-    implementation 'com.twelvemonkeys.imageio:imageio-bmp:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-hdr:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-icns:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-iff:3.10.1'
-    implementation 'com.twelvemonkeys.imageio:imageio-jpeg:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-pcx:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-pict:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-pnm:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-psd:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-sgi:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-tga:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-thumbsdb:3.10.1'
-    implementation 'com.twelvemonkeys.imageio:imageio-tiff:3.10.1'
-    implementation 'com.twelvemonkeys.imageio:imageio-webp:3.10.1'
-    // implementation 'com.twelvemonkeys.imageio:imageio-xwd:3.10.1'
-
-    implementation 'commons-io:commons-io:2.16.1'
-    implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0'
```

Comment:
damm just saw this, there is a breaking change for our pipeline stuff in 2.3.0 onwards which pipeline doesn't support yet

Reply:
will make a PR to change back

Reply:
sorry I didn't test this